src/components/ItemList.tsx

```diff
@@ -6,7 +6,7 @@ import useStore from "../store"
 
 const list = [
   { name: 10 },
-  { name: 20 },
+  { name: 30 }, // list with 30 items
```

Comment:
Ok, but currently we dont need this change and avoid unneccessary comment.

Reply:
Ok I will replace it again.